tests/torch/test_model_graph_manager.py

```diff
@@ -362,3 +364,39 @@ def test_update_fused_bias(model_cls):
         assert torch.all(torch.isclose(model.conv.bias, torch.tensor([0.3000, 1.3000])))
         assert torch.all(torch.isclose(model.bn.bias, torch.tensor([-1.0600, -3.6000])))
         assert torch.all(torch.isclose(model.conv.bias * model.bn.weight + model.bn.bias, ref_new_bias))
+
+
+class TestTargetDimForWeightCompression:
+
+    MM_SHAPES = (
+        (1, 1, 0, 0),  # [3] x [3]
+        (2, 1, 0, 0),  # [3, 4] x [4]
```

Comment:
As far I understand, minmax algorithm should quantize the second matrix using per-tensor scheme.